components/automate-ui/src/app/modules/team/team-details/team-details.component.spec.ts

```diff
@@ -84,7 +82,7 @@ describe('TeamDetailsComponent v2', () => {
       }
     },
```

Comment:
Change the test name just above from 'TeamDetailsComponent v2' to 'TeamDetailsComponent'.